modules/sb-install-service-catalog.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-service-catalog.adoc
+
+[id='sb-enable-service-catalog-{context}']
+= Enabling service catalog
+
+Service catalog is included by default in {product-title}, but you must you must
+enable it using the procedure below.
+
+.Procedure
+
+. Create the service catalog API server.
+.. Create a file named `svcat-api-server.yaml` with the following contents.
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: ServiceCatalogAPIServer
+metadata:
+  name: cluster
+spec:
+  logLevel: "Normal"
+  managementState: Managed
+----
+.. Use the following command to create the service catalog API server.
++
+----
+$ oc create -f svcat-api-server.yaml
```

Comment:
Similar to above with API Server, the ServiceCatalogControllerManager resource is created during install, but admins need to set the `managedStage` to `Managed`.  Change `oc create` to `oc apply`.

Reply:
@jboyd01 Actually a question - the files right now set "managementState" to Managed (for this and the API server. Is this correct, or should it be "managedStage" like your comments say?